src/aten/sycl/BinaryKernels.cpp

```diff
@@ -74,6 +74,41 @@ void div_kernel(TensorIteratorBase& iter) {
   }
 }
 
+template <typename scalar_t>
```

Comment:
Please move the change to corresponding file as stock CUDA, It should be `aten/src/ATen/native/cuda/BinaryMiscBackwardOpsKernels.cu`?